modules/currency-l0/src/main/scala/org/tessellation/currency/l0/CurrencyL0App.scala

```diff
@@ -60,6 +63,10 @@ abstract class CurrencyL0App(
       p2pClient = P2PClient.make[IO](sdkP2PClient, sdkResources.client, sdkServices.session)
       storages <- Storages.make[IO](sdkStorages, cfg.snapshot, method.globalL0Peer).asResource
       validators = Validators.make[IO](seedlist)
+      maybeMajorityPeerIds =
+        (if (sdk.realignWithNonPrioritySeedlist) sdk.seedlist else sdk.prioritySeedlist)
```

Comment:
This seedlist is a seedlist for the current layer not for the GL0. There is a `l0seedlistPath` in `CliMethod` in sdk module used for injecting the GL0 seedlist into stateChannelValidator (you can look at `TessellationIOApp`). It is hardcoded to be `None` for CL0, CL1 and DAGL1 disabling the l0seedlist checks during snapshot processing. If you want to use this l0seedlist for other purposes I guess you need to set it to a proper value for every layer using it. For GL0 it is === to seedlist (and set like that), but for CL0, CL1 and DAGL1 these seedlists may and most probably will be different. AFAIR DAGL1 currently runs without it's own seedlist.

Reply:
@TheMMaciek I don't understand -- is there a concern here? My understanding from @ryle-goehausen is that setting the `realignWithNonPrioritySeedlist` flag is for advanced setup, where the implementers will need to supply special scripts. The default is that we use the priority seed list.

Reply:
@TheMMaciek will it cause a problem on CL0/CL1/DAGL1 if there is a seed list passed through CLI the way david is mentioning? I.e. is the validation check on seedlist implemented at all of the others levels here or only for GL0? I.e. can we rely on user configuration here rather than creating another duplicate CLI param for secondary list?

Reply:
If the answer is no, then we need to provide a secondary CLI param for adjusting majority

Reply:
What I understand was the intention for this functionality is: "If the prioritySeedlist is not set use the peers from seedlist for the majority discovery". And I'm only pointing out that the `sdk.seedlist` used here is the seedlist for this particular layer a user is running - here it's CL0. I think that currently none of the layers use the seedlist so the value here will be empty list or None. And I'm pointing it out because I suspect what you wanted is for this seedlist to be the one on GL0, which it's not. If a CL0 owner sets a seedlist for the the layer with this configuration the `majorityPeers` will be the ones on the seedlist for this layer - I think it may checkout on the CL0 given that these nodes should also be on the GL0 seedlist, but it will not work for CL1 - there is no requirement for these to be on GL0 seedlist.

Anyway I don't think we actually need the `realignWithNonPrioritySeedlist` flag, if the user wants a different seedlist for majority and different for the layer then he/she needs to have to separate seedlists, if seedlist for the layer doesn't exist the user needs to specify the list for majority peers, if both list are the same then the user just needs to set the value of both seedlist and majorityPeers to the same source file and will get the same behaviour.

Reply:
As discussed in meeting and earlier comments: we don't need this parameter anymore since we don't have the GL0 seed list being passed in anywhere already -- so we can't use it as a fallback.